src/core/agent/agent.types.ts

```diff
@@ -18,7 +18,6 @@ enum ConnectionHistoryType {
 }
 
 enum ConnectionType {
```

Comment:
Lets remove this enum and remove any fields that have this connection type since we will only have one now